src/test/java/com/intel/gkl/pairhmm/PairHmmUnitTest.java

```diff
@@ -54,36 +56,71 @@ public void simpleTest() {
 
     @Test(enabled = true)
     public void omp_Test() {
-        final IntelPairHmm_omp pairHmm = new IntelPairHmm_omp();
-        final boolean isSupported = new IntelPairHmm_omp().load();
-        Assert.assertTrue(isSupported);
-
-        final PairHMMNativeArguments args = new PairHMMNativeArguments();
-        args.maxNumberOfThreads = 10;
-        args.useDoublePrecision = false;
-
-        pairHmm.initialize(args);
-
-        ReadDataHolder[] readDataArray = new ReadDataHolder[1];
-        HaplotypeDataHolder[] haplotypeDataArray = new HaplotypeDataHolder[1];
-        double[] likelihoodArray = new double[1];
-
-        // read data from file
-        haplotypeDataArray[0] = new HaplotypeDataHolder();
-        haplotypeDataArray[0].haplotypeBases = "ACGT".getBytes();
-        readDataArray[0] = new ReadDataHolder();
-        readDataArray[0].readBases = "ACGT".getBytes();
-        readDataArray[0].readQuals = "++++".getBytes();
-        readDataArray[0].insertionGOP = "++++".getBytes();
-        readDataArray[0].deletionGOP = "++++".getBytes();
-        readDataArray[0].overallGCP = "++++".getBytes();
-        double expectedResult = -6.022797e-01;
-
-        // call pairHMM
-        pairHmm.computeLikelihoods(readDataArray, haplotypeDataArray, likelihoodArray);
-
-        // check result
-        Assert.assertEquals(likelihoodArray[0], expectedResult, 1e-5, "Likelihood not in expected range.");
+        final boolean isSupported = new IntelPairHmmOMP().load();
+        if(!isSupported) {
+            final IntelPairHmm pairHmm = new IntelPairHmm();
+
+            final boolean isloaded = pairHmm.load(null,"gkl_pairhmm");
```

Comment:
The test should not pass the library name here either. GATK code is going to look something like this:

```java
boolean isSupported = new IntelPairHmmOMP().load();
if(!isSupported) {
   isSupported = new IntelPairHmm().load();
}
Assert.assertTrue(isSupported);
```